light-node/src/commands/start.rs

```diff
@@ -108,53 +120,59 @@ impl config::Override<LightNodeConfig> for StartCmd {
         &self,
         mut config: LightNodeConfig,
     ) -> Result<LightNodeConfig, FrameworkError> {
-        if !self.rpc_addr.is_empty() {
-            config.rpc_address = self.rpc_addr.to_owned();
+        // Todo figure out if other options would be reasonable to overwrite via CLI arguments.
+        if let Some(addr) = self.listen_addr {
+            config.rpc_config.listen_addr = addr;
         }
-
         Ok(config)
     }
 }
+impl StartCmd {
+    fn make_instance(
```

Comment:
Mentioned it in https://github.com/informalsystems/tendermint-rs/pull/431#discussion_r452653735 - there are enough occurrences to make this a library method,

Reply:
Yes, makes sense. I thought as part of the integration tests work with @Shivani912 there now was such a method but it seems that a slightly modified version is copied where needed currently. Can we address this separately?

Reply:
Already captured in #422 thanks @romac for flagging.